### ISSUE TITLE
Automatically create Listable variations for types that aren't already Listable

### ISSUE THREAD

**Issue by evaera** [evaera/Cmdr#58]

**Comment by LoganDark** [comment]
This could open up many possibilities for 'variant' types.

**Comment by ElliottLMz** [comment]
Would there be a chance of this causing problems for types that _really shouldn't_ be listable. E.g. PlayerID can be expensive if enough players are specified, or in some commands (e.g. place players [place id]) it could be potentially problematic.

**Comment by evaera** [comment]
> Would there be a chance of this causing problems for types that _really shouldn't_ be listable. E.g. PlayerID can be expensive if enough players are specified, or in some commands (e.g. place players [place id]) it could be potentially problematic.

No, I don't think so. This would create a plural type out of any singular types, so commands can still choose to use the singular variation if they wish which isn't listable. This change would only mean that if a type like "integer" is registered, it will automatically register "integers". But if you only wanted one integer, you could still use type "integer" in your command. So the control is still up to the developer here.
